infra/base-images/base-runner/Dockerfile

```diff
@@ -14,17 +14,24 @@
 #
 ################################################################################
 
-# Using multi-stage build to copy some LLVM binaries needed in the runner image.
-FROM gcr.io/oss-fuzz-base/base-clang AS base-clang
+# Using multi-stage build to copy some LLVM binaries and the Jazzer binaries
+# needed in the runner image.
+FROM gcr.io/oss-fuzz-base/base-builder AS base-builder
 
 FROM gcr.io/oss-fuzz-base/base-image
 
 # Copy the binaries needed for code coverage and crash symbolization.
-COPY --from=base-clang /usr/local/bin/llvm-cov \
+COPY --from=base-builder /usr/local/bin/llvm-cov \
      /usr/local/bin/llvm-profdata \
      /usr/local/bin/llvm-symbolizer \
      /usr/local/bin/
 
+# Copy the Jazzer binaries.
+COPY --from=base-builder /usr/local/bin/jazzer \
```

Comment:
It's not great that we depend on base-builder now to build base-runner.
I want to avoid surprises where one changes base-builder but because of the order in which images are built, base-runner isn't updated.
But I don't have a work around for this and I suppose the same issue existed with base-clang (though it's probably a bigger problem here).

Reply:
The only alternative I see would be to move the Jazzer build into `base-clang`. Jazzer includes bazelisk and builds with OpenJDK 8, so this wouldn't add a lot of complexity to `base-clang`.

Reply:
I think there's another solution here.
OSS-Fuzz uses fuzzers like AFL and honggfuzz where the fuzzer (e.g. afl-fuzz) is a separate binary from the fuzz target (i.e. the program that gets input from the fuzzer).
For those fuzzers, OSS-Fuzz includes the fuzzer in the $OUT directory along with the targets.
I think that's the right way to do things for jazzer.
Putting jazzer in the $OUT directory will ensure that we are always fuzzing a fuzz target with the correct fuzzer.

Reply:
I'm now copying Jazzer into `/usr/local/bin` and then moving it into `$OUT` in `compile`.